src/tag-editor/__tests__/tag-editor.test.tsx

```diff
@@ -569,7 +571,13 @@ describe('Tag Editor component', () => {
 
       wrapper.findRow(1)!.findRemoveButton()!.click();
       rerender();
-      const nextInput = wrapper.findRow(1)!.findAutosuggest()!.findNativeInput().getElement();
+      const nextInput = wrapper
+        .findRow(1)!
+        .findField(1)!
+        .findControl()!
+        .findAutosuggest()!
+        .findNativeInput()
+        .getElement();
```

Comment:
Bonus effect of this improvement. Now we strictly target the first column in a row and not an autosuggest in any column